shard.yml

```diff
@@ -34,6 +34,7 @@ dependencies:
     github: stefanwille/crystal-redis
     version: ~> 1.9.0
 
+development_dependencies:
   cli:
```

Comment:
So, I need to remove `--production` flag [here](https://github.com/amberframework/amber-aur/blob/master/PKGBUILD#L15), Is the cli a development dependency only?

Reply:
@faustinoaq Yes, the CLI should be used for development only.  

Is the `amber` command required for heroku deployment?  If so, can we use the docker image instead?

Reply:
@faustinoaq just remembered to run the migrations in production, we need the amber cli.  Lets hold off on this PR until we figure out how to do deployments without requiring to build `amber`.